extensions/wikia/CreateNewWiki/maintenance/setMainPageContent.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/**
+ * This script updates the main page content adding wiki name and description provided by the founding user.
+ */
+
+require_once( __DIR__ . '/../../../../maintenance/Maintenance.php' );
+
+class SetMainPageContent extends Maintenance {
+
+	public function execute() {
+		global $wgWikiDescription, $wgCityId;
+
+		$this->output( 'SetMainPageContent: Started' );
+		$this->output( 'SetMainPageContent: description is ' . $wgWikiDescription );
+		if ( empty( $wgWikiDescription ) ) {
+			$this->output( 'SetMainPageContent: Empty wiki description, skipping' );
+			return;
+		}
+
+		// this runs before the StartPostCreationTasks so main page is available under the old address
+		// @see: CreateNewWikiTask::moveMainPage
+		$mainTitle = Title::newFromText( wfMsgForContent( 'Mainpage' ) );
```

Comment:
`wfMsgForContent` might not work consistently, see https://github.com/Wikia/app/pull/10648. This might not be the case anymore after flow changes, I'm not sure.

Reply:
The order of tasks is different now so it's no longer an issue.